### ISSUE TITLE
Permission denied. ... File "chia\plotting\create_plots.py", line 176, in create_plots RuntimeError: Unknown exception [BUG]

### ISSUE THREAD

**Issue by LikotUbendeb** [Chia-Network/chia-blockchain#3725]
I tried many plotting configuration, the error allways the same: **RuntimeError: Unknown exception** What am i supposed to do???

2021-05-05T00:21:30.363  chia.plotting.create_plots       : [32mINFO    [0m Creating 1 plots of size 32, pool public key:  9154de61f553e0c79ad3b86f5417293cbdbbaf26d7e2c19af817ef1391228a0df355a644f9de2f53864abefe0d64aaa6 farmer public key: b2a0178c5c52b227407ecc42e1f2b17520a19c3520712c88c8654fa2fa53d717815d87de69c2b4e081daad88a826ff1d[0m
2021-05-05T00:21:30.371  chia.plotting.create_plots       : [32mINFO    [0m Memo: 9154de61f553e0c79ad3b86f5417293cbdbbaf26d7e2c19af817ef1391228a0df355a644f9de2f53864abefe0d64aaa6b2a0178c5c52b227407ecc42e1f2b17520a19c3520712c88c8654fa2fa53d717815d87de69c2b4e081daad88a826ff1d4b74b56b26bd0d71439a40a311e68f606ebff321339c9462035c66e93f6691ac[0m
2021-05-05T00:21:30.372  chia.plotting.create_plots       : [32mINFO    [0m Starting plot 1/1[0m

Starting plotting progress into temporary dirs: R:\ and R:\
ID: 6b08532d4b96be7ed37ca740eb8c591642b394f3891010420f0806ca8b64669e
Plot size is: 32
Buffer size is: 6000MiB
Using 128 buckets
Using 4 threads of stripe size 65536
Caught plotting error: Could not open R:\plot-k32-2021-05-05-00-21-6b08532d4b96be7ed37ca740eb8c591642b394f3891010420f0806ca8b64669e.plot.sort.tmp: Permission denied.
[14060] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 176, in create_plots
RuntimeError: Unknown exception

**Comment by LikotUbendeb** [comment]
I upgraded from 16 gb ram to 32. would it be important?

**Comment by LikotUbendeb** [comment]
Also i have rx 570 that mining eth at the same PC, and my CP is ryzen 5 2600 overclocked 4000. My power unit has 400 wt. Also i want to say that before upgrate my ram worked at 3200 mhz, but know pc shuts down after 2800 mhz.

**Comment by LikotUbendeb** [comment]
I found the reason

**Comment by LikotUbendeb** [comment]
You MUST make a folder in the disk, you can't just select C:/ D:/ B:/. Select С:/TempFolder D:/PlotFolder and so on

I hope You fix this

**Comment by LikotUbendeb** [comment]
in 90% of cases, the problem was solved when I created an additional folder in the disk, that is, I chose not C:/, C:/Temp/ for example.
So wierd... 

also I pulled out 2 ram. Now i'm going to stick them back.

**Comment by LikotUbendeb** [comment]
So yeah. Just changing repository solves the problem. If not just try again, i guess.

**Comment by MikhailZelenkov** [comment]
Very similar to RAM problems.

**Comment by LikotUbendeb** [comment]
> 
> 
> Very similar to RAM problems.

May be but it's not obvious. My solution of the problem is selecting folder and NOT disk partition's root. Hope it'll help someone.

**Comment by macm1972** [comment]
same trouble -----Caught plotting error: Could not open :\plot-k32-2021-00-00-00-00-6b08532d4b96be7ed37ca740eb8c591642b394f3891010420f0806ca8b64669e.plot.sort.tmp: Permission denied.-----

can not solve it. Sorry i am unexperience in this chia-blockchain.

**Comment by Plopsi** [comment]
Windows? are you administrator? Use a disk formatted with a FS which has no user-permissions, like exFat. Or use a user which has accessrights, as you mentioned for example directly C:\ is an area with special permission needs.

In your last comment the drive-letter is missing (accident?), did the drive disconnect? Otherwise, while very unlikely, do a RAM-test - suddenly missing values are no good sign.

**Comment by macm1972** [comment]
Sorry, is Linux. and can not to watch any letter. P.E dev is the* ssd *and
sda is *hdd. *

**Comment by bendybones** [comment]
same issue here on both plotting drives (ubuntu)
any idea how to fix this?

Starting plotting progress into temporary dirs: /media/harvester1/plotting and /media/harvester1/plotting
ID: 60739fa95562d9f661aac23aae43b9104b6fa4d5714a4b3767dd38e7bba1c24b
Plot size is: 32
Buffer size is: 3390MiB
Using 128 buckets
Using 2 threads of stripe size 65536
Caught plotting error: Could not open /media/harvester1/plotting/plot-k32-2021-05-09-20-32-60739fa95562d9f661aac23aae43b9104b6fa4d5714a4b3767dd38e7bba1c24b.plot.sort.tmp: Permission denied.
Traceback (most recent call last):
  File "chia/cmds/chia.py", line 81, in <module>
  File "chia/cmds/chia.py", line 77, in main
  File "click/core.py", line 829, in __call__
  File "click/core.py", line 782, in main
  File "click/core.py", line 1259, in invoke
  File "click/core.py", line 1259, in invoke
  File "click/core.py", line 1066, in invoke
  File "click/core.py", line 610, in invoke
  File "click/decorators.py", line 21, in new_func
  File "chia/cmds/plots.py", line 135, in create_cmd
  File "chia/plotting/create_plots.py", line 164, in create_plots
RuntimeError: std::exception
[22119] Failed to execute script chia

**Comment by dogan2893** [comment]
Everybody listen to me:
All you have to do is create a new file on local disk C and local disk D yourself.
Important: Do not try to create a new folder from the create plot page.
Nothing to do with permission I tried it

**Comment by macm1972** [comment]
Thanks, I'll check it out.

**Comment by CDRohling** [comment]
I fixed it by formating my harddrive to NTFS, as exfat is not supporting any permissions.

**Comment by Plopsi** [comment]
exFAT also doesn't support trimming which comes in very handy on SSDs and performance - so yes, always use NTFS or ReFS on Windows

**Comment by macm1972** [comment]
Thanks for the tip!

**Comment by eagle283** [comment]
I had this same issue today.  I'm running Win 10 and had 5 plots stuck because of permission denied.  I looked at the log of the first plot that was stuck, found the file that had permission issues and copied the name to notepad.  I went to my ssd temp plotting directory and searched for the file name and copied that file to a different directory, deleted the old one in the temp plotting directory and than copied it back to the plotting directory.  All of the plots that were stuck are now continuing.  I hope this helps you.

**Comment by eagle283** [comment]
I had this same issue today. I'm running Win 10 and had 5 plots stuck because of permission denied. I looked at the log of the first plot that was stuck, found the file that had permission issues and copied the name to notepad. I went to my ssd temp plotting directory and searched for the file name and copied that file to a different directory, deleted the old one in the temp plotting directory and than copied it back to the plotting directory. All of the plots that were stuck are now continuing. I hope this helps you.
